src/library/Uno.Material/Styles/Controls/CommandBar.xaml

```diff
@@ -17,18 +17,25 @@
 
 	<Style x:Key="MaterialAppBarButton"
 		   TargetType="AppBarButton">
-		<Setter Property="Height" Value="{StaticResource MaterialCommandBarHeight}" />
-		<Setter Property="Width" Value="{StaticResource MaterialCommandBarHeight}" />
-		<Setter Property="Background" Value="Transparent" />
-		<Setter Property="Foreground" Value="{StaticResource MaterialOnPrimaryBrush}" />
-		<Setter Property="HorizontalAlignment" Value="Center" />
-		<Setter Property="VerticalAlignment" Value="Center" />
-		<Setter Property="IsCompact" Value="True" />
+		<Setter Property="Height"
```

Comment:
2-attributes setters shouldn't be on multiple lines

Reply:
generally... when im working on a xaml file, unless i created it, i usually keep the existing setup (same line or 2lines) to make the changes minimal
unless im doing major refactor on them, that also touch heavily the setters
like ive said before, you can always xaml-format and then use "revert selected ranges" to undo what should not be touched

Reply:
Format changes removed.
We will be able to decide what to do with xaml formatting guidelines later on.